src/chainparams.cpp

```diff
@@ -139,6 +139,9 @@ class CMainParams : public CChainParams {
         consensus.vUpgrades[Consensus::UPGRADE_NU5].nActivationHeight = 1687104;
         consensus.vUpgrades[Consensus::UPGRADE_NU5].hashActivationBlock =
             uint256S("0000000000d723156d9b65ffcf4984da7a19675ed7e2f06d9e5d5188af087bf8");
+        consensus.vUpgrades[Consensus::UPGRADE_NU6].nProtocolVersion = 0xC8E71055;
```

Comment:
We got completely confused earlier. This is the network protocol version, not the consensus branch ID.
```suggestion
        consensus.vUpgrades[Consensus::UPGRADE_NU6].nProtocolVersion = 170120;
```